scripts/run_local.sh

```diff
@@ -0,0 +1,170 @@
+# - Service Catalog needs to talk to route and have it reach the local broker 
+#   - Update the asb service & endpoint to point to our local broker 
+# - Create a route for etcd so local broker can talk to etcd
+# - Generate a configuration file for broker to use
+# - Set KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT to reach cluster 
+# - Get existing secret for the asb service account and store ca cert/token
+#
+###
+
+MY_VARS="./my_local_dev_vars"
+if [ ! -f $MY_VARS ]; then 
+  echo "Please create $MY_VARS"
+  echo "cp $MY_VARS.example $MY_VARS"
+  echo "then edit as needed"
+  exit 1
+fi 
+source ./my_local_dev_vars
+
+BROKER_CMD="../broker"
+GENERATED_BROKER_CONFIG="../etc/generated_local_development.yaml"
+TEMPLATE_LOCAL_DEV="../templates/deploy-local-dev-changes.yaml"
+ASB_PROJECT="ansible-service-broker"
+BROKER_SVC_ACCT_NAME="asb"
+BROKER_SVC_ACCT="system:serviceaccount:${ASB_PROJECT}:${BROKER_SVC_ACCT_NAME}"
+
+# Faking out https://github.com/kubernetes/client-go/blob/master/rest/config.go#L309
+export KUBERNETES_SERVICE_HOST=${OPENSHIFT_SERVER_HOST}
+export KUBERNETES_SERVICE_PORT=${OPENSHIFT_SERVER_PORT}
+SVC_ACCT_TOKEN_DIR=/var/run/secrets/kubernetes.io/serviceaccount
+SVC_ACCT_CA_CRT=$SVC_ACCT_TOKEN_DIR/ca.crt
+SVC_ACCT_TOKEN_FILE=$SVC_ACCT_TOKEN_DIR/token
+
+# We rely on jq for parsing json data from oc/kubectl
+which jq &> /dev/null
+if [ "$?" -ne 0 ]; then 
+  echo "Please ensure 'jq' is installed and in your path"
+  exit 1
+fi 
+
+# We will fake out the service account directory locally on the machine
+# The directory is under /var/run and likely to be deleted between reboots
+if [ ! -d "$SVC_ACCT_TOKEN_DIR" ]; then 
+  echo "Attempting to create serviceaccount directory: ${SVC_ACCT_TOKEN_DIR}"
+  sudo mkdir -p ${SVC_ACCT_TOKEN_DIR}
+  if [ "$?" -ne "0" ]; then 
+    echo "Please create serviceaccount directory with read/write permissions for your user:  ${SVC_ACCT_TOKEN_DIR}"
+    exit 1
+  fi 
+fi 
+sudo chown ${USER} ${SVC_ACCT_TOKEN_DIR}
+if [ "$?" -ne "0" ]; then 
+  echo "Please chown the serviceaccount directory so your user may read/write:  ${SVC_ACCT_TOKEN_DIR}"
+  exit 1
+fi 
+ 
+
+# Determine the name of the secret which has the 'asb' service account info
+BROKER_SVC_ACCT_SECRET_NAME=`oc get serviceaccount asb -n ansible-service-broker -o json | jq -c '.secrets[] | select(.name | contains("asb-token"))' | jq -c '.name'`
+# Remove quotes from variable
+BROKER_SVC_ACCT_SECRET_NAME=( $(eval echo ${BROKER_SVC_ACCT_SECRET_NAME[@]}) )
+echo "Broker Service Account Token is in secret: ${BROKER_SVC_ACCT_SECRET_NAME}"
+
+###
+# Fetch the service-ca.crt for the service account
+###
+SVC_ACCT_CA_CRT_DATA=`oc get secret ${BROKER_SVC_ACCT_SECRET_NAME} -n ${ASB_PROJECT} -o json | jq -c '.data["service-ca.crt"]'`
+# Remove quotes from variable
+SVC_ACCT_CA_CRT_DATA=( $(eval echo ${SVC_ACCT_CA_CRT_DATA[@]}) )
+# Base64 Decode
+SVC_ACCT_CA_CRT_DATA=`echo ${SVC_ACCT_CA_CRT_DATA} | base64 -D `
```

Comment:
```echo ${SVC_ACCT_CA_CRT_DATA} | base64 --decode ``` will work on both Linux and Darwin.  It appears that linux wants -d and darwin wants -D